src/main/java/de/cityfeedback/feedbackverwaltung/logging/LoggingAspect.java

```diff
@@ -0,0 +1,24 @@
+package de.cityfeedback.feedbackverwaltung.logging;
```

Comment:
Die Klasse müsste laut chatgpt im domaindriven design in in feedbackverwaltung.infrastructure.aspects einsortiert werden